src/integTest/kotlin/com/ampnet/auditornode/persistence/repository/LocalIpfsRepositoryIntegTest.kt

```diff
@@ -0,0 +1,213 @@
+package com.ampnet.auditornode.persistence.repository
+
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import assertk.assertions.isInstanceOf
+import com.ampnet.auditornode.IntegTestBase
+import com.ampnet.auditornode.IntegTestUtils
+import com.ampnet.auditornode.isLeftSatisfying
+import com.ampnet.auditornode.isRightContaining
+import com.ampnet.auditornode.isRightSatisfying
+import com.ampnet.auditornode.model.error.IpfsError.IpfsHttpError
+import com.ampnet.auditornode.persistence.model.IpfsHash
+import com.ampnet.auditornode.persistence.model.IpfsTextFile
+import com.ampnet.auditornode.persistence.repository.impl.LocalIpfsRepository
+import com.ampnet.auditornode.testcontainers.IpfsTestContainer
+import com.ampnet.auditornode.testcontainers.IpfsTestContainer.uploadFileToIpfs
+import com.ampnet.auditornode.testcontainers.IpfsTestContainer.uploadFileToIpfsDirectory
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest
+import io.micronaut.test.support.TestPropertyProvider
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import javax.inject.Inject
+
+@MicronautTest
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class LocalIpfsRepositoryIntegTest : IntegTestBase(), TestPropertyProvider {
```

Comment:
The same as for class `GatewayIpfsRepositoryIntegTest`, maybe before each, see what suits you better.